commands/livecmd.go

```diff
@@ -78,10 +79,15 @@ func GetLiveCommand(name string, f util.Factory) *cobra.Command {
 	destroyCmd.Long = livedocs.DestroyShort + "\n" + livedocs.DestroyLong
 	destroyCmd.Example = livedocs.DestroyExamples
 
+	statusCmd := status.StatusCommand(f)
+	statusCmd.Short = "Show status of a package already installed in the cluster"
```

Comment:
Shouldn't these refer to the variables in the generated docs?

Reply:
Yeah, updating the docs and generating these docs are in #1014

Reply:
So this PR should be merged after the rebase ?